docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - "5343:5343"
   redis:
     image: redis
+#    command: redis-server --requirepass redispassword
```

Comment:
This command is only necessary, when we want to use password on redis-server.
Otherwise it may be commented out.
Would it be better to specify in the documentation?